web/libs/datamanager/src/utils/dom.js

```diff
@@ -70,7 +70,7 @@ export const alignElements = (elem, target, align, padding = 0) => {
   if (offsetTop < window.scrollX) {
     offsetTop = pos.bottom + padding;
     resultAlign[0] = "bottom";
-  } else if (offsetTop + pos.target.height > window.scrollX + window.innerHeight) {
+  } else if (openUpwardForShortViewport && offsetTop + pos.target.height > window.scrollX + window.innerHeight) {
```

Comment:
I see a huge bug here with wrong dimensions in old code — "top" is compared to "x" instead of "y". The main reason I guess it didn't cause troubles is that dropdowns are used in a non-scrollable areas maybe. But it should be fixed.